### DISCUSSION TITLE
Could we make GUI work with multiple repositories?

### DISCUSSION THREAD

**Discussion by jacobgexigo** [trimble-oss/dba-dash#136] [Ideas]
Just wondering if you could add the capability to attach the GUI to multiple repositories and just switch between them?  The thought being if you had internal resources you wanted to monitor and also contract work servers to monitor and just wanted to keep them separate at the repository level (for many reasons).

**Comment by DavidWiseman** [comment]
Yeah - this is something I've thought about.  Currently, the only option is deploying multiple copies of the GUI and launching the internal version or the contract version as required.  This is a bit clunky - it would be nice to have an option to switch between repositories. 
I might consider adding this at some point as I can see some potential benefits for Trimble also.  I'll create an issue from this discussion.

**Comment by DavidWiseman** [comment]
Fast switching between repositories is now available in [2.40.0](https://dbadash.com/blog/whats-new-in-2.40.0/ )  🎉
